docs/source/tutorial/running_ablation.rst

```diff
@@ -1,3 +1,87 @@
 Running an Ablation Study
--------------------------
-.. automodule:: pykeen.ablation
+=========================
+In PyKEEN, we can define an ablation study in a configuration file ('file_name.json') or within our own program.
+
+First, we show how to define an ablation study within a configuration file. In the following example, we define an
+ablation study for ComplEx over the Nations dataset, and name the file 'complex_nation.json'.
+In particular, we want to asses the effect of different loss functions, i.e., the binary cross entropy loss
+and the margin ranking loss, functions, and the effect of explicitly modeling inverse relations.
+
+We start by adding metadata to our configuration file. This is helpful for a later retrospection.
+
+.. code-block:: javascript
+
+    {
+        "metadata": {
+            "title": "Ablation Study Over Nations for ComplEx."
+        }
+    }
+
+Now, we start with the actual definition of the ablation study, and define the minimal requirements, i.e.,
+the dataset(s), interaction model(s), the loss function(s), training approach(es), and the optimizer(s):
+
+.. code-block:: javascript
+
+    {
+        "metadata": {
+            "title": "Ablation Study Over Nations for ComplEx."
+        },
+        "ablation": {
+            "datasets": ["nations"],
+            "models":   ["ComplEx"],
+            "losses": ["BCEAfterSigmoidLoss", "CrossEntropyLoss"]
+            "training_loops": ["lcwa"]
+        }
+    }
+
+However, we also want to measure the influence of explicitly modeling inverse relations. Therefore,
+we extend the ablation study accordingly:
+
+.. code-block:: javascript
+
+    {
+        "metadata": {
+            "title": "Ablation Study Over Nations for ComplEx."
+        },
+        "ablation": {
+            "datasets": ["nations"],
+            "models":   ["ComplEx"],
+            "losses": ["BCEAfterSigmoidLoss", "CrossEntropyLoss"]
+            "training_loops": ["lcwa"],
+            "create_inverse_triples": [true,false]
+        }
+    }
+
+For each of the components of a knowledge graph embedding model (KGEM) that requires hyper-parameters, i.e.,
```

Comment:
what's the difference between ablation study and HPO? probably will be covered in the first few graphs but just in case, do a short refresher